openmetadata-docs/content/v1.7.x-SNAPSHOT/getting-started/day-1/hybrid-saas/hybrid-ingestion-runner.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Hybrid Ingestion Runner
+slug: /getting-started/day-1/hybrid-saas/hybrid-ingestion-runner.md
+collate: true
+---
+
+# Hybrid Ingestion Runner
+
+The **Hybrid Ingestion Runner** is a component designed to enable Collate customers operating in hybrid environments to securely execute ingestion workflows within their own cloud infrastructure. In this setup, your SaaS instance is hosted on Collate’s cloud, while tools like Argo Workflows are deployed within your private cloud. The Hybrid Runner acts as a bridge between these two environments, allowing ingestion workflows to be triggered and managed remotely—without requiring the customer to share secrets or sensitive credentials with Collate. It securely receives workflow execution requests and orchestrates them locally, maintaining full control and data privacy within the customer’s environment.
+
+## Prerequisites
+
+Before setting up the Hybrid Ingestion Runner, ensure the following:
+
+- Your Collate SaaS instance is available.
+- You have **[Argo Workflows](https://argoproj.github.io/workflows/)** deployed in your own cloud environment.
+- Network access between your own cloud and Collate SaaS server (outbound traffic from your cloud to Collate).
+- Java 17+ for the runner.
+- Docker.
```

Comment:
end users don't rly need java, docker, or care about the network. everything is UI based